### PR TITLE
Updated action names for adding repo

### PR DESCRIPTION
**Pull request by bartaz**
Reimplements #571 

Updating add repository action names. 
Took a chance and did some repository/repo renaming on the way.

Part of #541

### CODE REVIEW

File: src/common/actions/repository.js [hunk 1]
```diff
@@ -36,12 +36,13 @@ export function buildRepositories(repositories, owner) {
   };
 }
 
-export function buildRepository(repository) {
+// add a repository to launchpad's build queue, returns a snap representation
```
Comment (earnubs) [inline src/common/actions/repository.js]: the fetch might, but we're not handling it here are we? it's just updating the original repository state i think

Comment (bartaz) [inline src/common/actions/repository.js]: yep, ok - let's not make the comment more confusing that it needs to be :)

File: test/unit/src/common/actions/t_repositories.js [hunk 2]
```diff
@@ -132,7 +132,7 @@ describe('repositories actions', () => {
       it('should store no pageLinks', async () => {
         await store.dispatch(fetchUserRepositories());
         expect(store.getActions()).notToHaveActionOfType(
-          ActionTypes.SET_REPOSITORY_PAGE_LINKS
+          ActionTypes.SET_REPO_PAGE_LINKS
```
Comment (earnubs) [inline test/unit/src/common/actions/t_repositories.js]: not sure this exists anymore, but the test hasn't been removed or updated so it's a valid change here

Comment (bartaz) [inline test/unit/src/common/actions/t_repositories.js]: It probably doesn't exist, because I think I only seen it in this one place in test when renaming.

**Pull request merged**